src/main/java/net/snowflake/ingest/streaming/internal/AbstractRowBuffer.java

```diff
@@ -274,7 +274,7 @@ Set<String> verifyInputColumns(
   @Override
   public InsertValidationResponse insertRows(
       Iterable<Map<String, Object>> rows, String offsetToken) {
-    float rowSize = 0F;
+    float rowsSize = 0F;
```

Comment:
Nit, maybe rename to `rowsSizeBytes` so we know what the unit is that we're tracking.